roles/kubernetes-apps/ansible/templates/coredns-deployment-secondary.yml.j2

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: coredns-secondary
+  namespace: {{ system_namespace }}
+  labels:
+    k8s-app: coredns-secondary
```

Comment:
Here and everywhere else you append -secondary in both the deployment and service, replace that with a jinja2 filter, something like 
```yaml
  name: coredns{{ coredns_ordinal_suffix | default(omit) }}
```
With that done, you could just use the single template with dynamic names.